modules/distribution/resources/operation_policies/specifications/ccRewriteResourcePath_v1.json

```diff
@@ -0,0 +1,36 @@
+{
+  "category": "Mediation",
+  "name": "rewriteResourcePath",
+  "version": "v1",
+  "displayName": "Rewrite Resource Path",
+  "description": "This policy allows you to rewrite the resource path of a request",
+  "policyAttributes": [
+    {
+      "name": "newResourcePath",
+      "displayName": "New Resource Path",
+      "description": "Name of the new resource path",
+      "validationRegex": "^\\/?(({uri\\.var\\.[\\w\\/\\-]+})|[\\w\\/\\-\\s])+$",
+      "type": "String",
+      "required": true
+    },
+    {
+      "name": "includeQueryParams",
+      "displayName": "Include Query Parameters",
+      "description": "Select whether to include exiting query params to the new resource path",
+      "validationRegex": "^(true|false)$",
+      "type": "Boolean",
+      "defaultValue": true,
+      "required": false
+    }
+  ],
+  "applicableFlows": [
+    "request"
+  ],
+  "supportedGateways": [
+    "Synapse",
```

Comment:
If this is only for CC, do we need to have Synapse here?

Reply:
using the same synapse policy, since it will support {uri.var.petID} format with the PR https://github.com/wso2/product-apim/pull/12670